superset/Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN microdnf update \
     && microdnf clean all && \
     rm -rf /var/cache/yum
 
+# DL3042 gets ignored, since we use --no-chache-dir, hadolint can't interfere properly
+# SC2102 informs about [abba] = [ab], but we can igonre this for [sqlalchemy] https://www.shellcheck.net/wiki/SC2102
+# hadolint ignore=DL3042,SC2102
```

Comment:
```suggestion
# DL3042 false-positive, --no-chache-dir is specified a few lines below.
#        See https://github.com/hadolint/hadolint/issues/1042.
# hadolint ignore=DL3042
```

Reply:
Re: `SC2102`, We should instead quote the offending line, because if globbing is enabled, it will break.

Like:

```diff
- trino[sqlalchemy] \
+ 'trino[sqlalchemy]' \
```